Library/Homebrew/cmd/configure-shell.rb

```diff
@@ -0,0 +1,53 @@
+#:  * `configure-shell`:
+#:    Configure the shell to include Homebrew into your PATH, MANPATH, and INFOPATH.
+#:    A brew.env file will be created that will be sourced in your ~/.profile
+#:    (or your .bash_profile or .zprofile - only existing files will be modified)
+
+module Homebrew
+  module_function
+
+  def configure_shell
+    # Ensure the directory for the brew.env file exists
+    FileUtils.mkdir_p HOMEBREW_PREFIX/"etc"
+
+    # Contents for the brew.env file
+    brew_env = <<~EOS
+      # Homebrew environment
+      # Created by running brew configure-shell
+      export PATH="#{HOMEBREW_PREFIX}/bin:#{HOMEBREW_PREFIX}/sbin:$PATH"
+      export MANPATH="#{HOMEBREW_PREFIX}/share/man:$MANPATH"
+      export INFOPATH="#{HOMEBREW_PREFIX}/share/info:$INFOPATH"
```

Comment:
Shall we add Homebrew bash completions here? 
```
. ${HOMEBREW_PREFIX}/Homebrew/completions/bash/brew
```

Reply:
Ah, nevermind - this is for `~/.profile`, not `~/.bashrc`...

Reply:
Hmm. I didn't know about this feature of Homebrew in fact. Let's merge this PR as is. We could consider completions in a separate PR.
Loading completions can't be done in `/home/linuxbrew/.linuxbrew/etc/brew.env` because that file is loaded by both `bash` and `zsh`. It could be added directly to `~/.bash_profile` and `~/.zprofile`.

Reply:
I believe `oh-my-zsh` and `prezto`, which are pretty common for zsh users, already have Homebrew completion enabled.

Reply:
> Ah, nevermind - this is for ~/.profile, not ~/.bashrc

Just saw this post after I wrote my response.

Reply:
No problem. I forgot this is for `.profile` files -- these are for "login" shells so should not have stuff like completions.

Reply:
Should this be resolved?

Reply:
I consider it resolved.